spec/result_spec.rb

```diff
@@ -5,15 +5,29 @@
 include Twingly::Search
 
 describe Result do
-  it { should respond_to :posts }
-  it { should respond_to :number_of_matches_returned }
-  it { should respond_to :number_of_matches_total }
-  it { should respond_to :seconds_elapsed }
-  it { should respond_to :all_results_returned? }
-  it { should respond_to :incomplete? }
+  instance_methods = [
+    :posts,
+    :number_of_matches_returned,
+    :number_of_matches_total,
+    :seconds_elapsed,
+    :all_results_returned?,
+    :incomplete?,
+  ]
+
+  instance_methods.each do |method|
+    it { should respond_to method }
+  end
 
   context "before query has populated responses" do
     its(:posts) { should be_empty }
     its(:all_results_returned?) { should be_truthy }
   end
+
+  describe "#inspect" do
+    subject { described_class.new.inspect }
+
+    instance_methods.each do |method|
+      it { should include(method.to_s) }
```

Comment:
Same as https://github.com/twingly/twingly-search-api-ruby/pull/83/files#r269577064 (it's not a problem right now though as all instance methods here have unique names)

Reply:
1c7200e